.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 python:
-  - "2.6"
-  - "2.7"
+  - "3.5"
+  - "3.6"
```

Comment:
We have 3.7 in Fedora. Please test it as well.

Reply:
Code is working on `py3.7 `
I guess Travis doesn't have it: https://travis-ci.org/sundeep-co-in/zanata-python-client/jobs/502504976 hence, skipping for now.

Reply:
thanks @hroncok